yarn.lock

```diff
@@ -1564,7 +1564,7 @@
   dependencies:
     "@endo/env-options" "^1.1.2"
 
-"@endo/exo@^1.3.0":
+"@endo/exo@^1.1.0", "@endo/exo@^1.3.0":
```

Comment:
See if this goes away once package.json dependency is fixed.

Reply:
Was indeed stale. Now that it is updated, the `yarn.lock` changes also disappeared. Done.